packages/components/src/components/menu-flyout/menu-flyout.tsx

```diff
@@ -126,7 +126,7 @@ export class MenuFlyout {
   }
 
   closeAll() {
```

Comment:
I would additionally make this an "arrow" function, just like the `toggle` below…

```diff
-closeAll() {
+closeAll = () => {
```

if it's really a `this` problem, this would probably fix it (pun intended haha)